common/app/views/fragments/items/facia_cards/contentCard.scala.html

```diff
@@ -28,11 +28,14 @@
     @item.snapStuff.map(_.dataAttributes)
     @item.shortUrl.map { shortUrl => data-loyalty-short-url="@shortUrl" }>
 
-    @if(!item.hasInlineSnapHtml) {
-        @container(item)
-    }
     @if(item.hasInlineSnapHtml) {
-        @item.snapStuff.map { snap => @snap.embedHtml.map(Html(_)) }
+        @item.snapStuff.map { snap =>
+            @snap.embedCss.map { css => <style>@Html(css)</style> }
```

Comment:
Is there any reason to prefer `@HtmlFormat.raw(...)` here instead of `@Html(...)`?

Reply:
Looks to me like they might be the same thing
https://www.playframework.com/documentation/2.0/api/scala/play/api/templates/HtmlFormat$.html

https://www.playframework.com/documentation/2.7.x/ScalaTemplates

I'd argue that htmlformat.raw sounds a bit more explicit that we're not escaping it but we use @Html more frequently so maybe makes sense to be consistent